### DISCUSSION TITLE
Where is the largest man-made swimming pool on earth?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#771] [Galaxy Trivia] [answered]
**Bonus:** What is the area of it?

**Comment by peterbaccaro** [comment, marked as answer]
The largest manmade swimming pool is **Lagoon Lake in Boulevard World** (Riyadh, Kingdom of Saudi Arabia) which opened on the 21st November 2022 .
It has an area of **12.19 hectares (30.13 acres)** and a height of 33.7 meters.

https://www.guinnessworldrecords.com/world-records/105697-largest-man-made-lagoon

**Top 5 Largest Manmade Swimming Pools**

1. Lagoon Lake in Boulevard World (Riyadh, Kingdom of Saudi Arabia) - 12.19 hectares - Opened in 2022
2. Citystars Sharm El Sheikh (Sharm El Sheikh, Egypt) - 9.68 hectares - Opened in 2015
3. Crystal Lagoon (Algarrobo, Chile) - 8 hectares - Opened in 2006
4. Mahasamutr (Hua Hin, Thailand) - 7.2 hectares - Opened in 2014
5. Treasure Bay Bintan (Bintan, Indonesia) - 6.3 hectares - Opened in 2015

**Comment by SANKARAMDAS** [comment]
San Alfonso del Mar, Chile
Apparently the pool is spread over 20 acres of land cost billions of dollars to build (easy to see why). The pool is in  Algarrobo in Chile.
